plugins/SliceInfoPlugin/MoreInfoWindow.qml

```diff
@@ -0,0 +1,158 @@
+            if (baseDialog.allowSendData)
+            {
+                allowSendButton.checked = true;
+            }
+            else
+            {
+                dontSendButton.checked = true;
+            }
+        }
+    }
+
+    Item
+    {
+        Connections
+        {
+            target: CuraApplication.getCuraAppSignals()
+            onShowMoreInfoOnAnonymousDataCollection:
+            {
+                baseDialog.show();
+            }
+        }
+
+        id: textRow
+        anchors
+        {
+            top: parent.top
+            bottom: radioButtonsRow.top
+            bottomMargin: UM.Theme.getSize("default_margin").height
+            left: parent.left
+            right: parent.right
+        }
+
+        Label
+        {
+            id: headerText
+            anchors
+            {
+                top: parent.top
+                left: parent.left
+                right: parent.right
+            }
+
+            text: catalog.i18nc("@text:window", "Cura sends anonymous data so we can improve the print quality and user experience. Below is an example of all the data we send.")
+            wrapMode: Text.WordWrap
+        }
+
+        TextArea
+        {
+            id: exampleData
+            anchors
+            {
+                top: headerText.bottom
+                topMargin: UM.Theme.getSize("default_margin").height
+                bottom: parent.bottom
+                bottomMargin: UM.Theme.getSize("default_margin").height
+                left: parent.left
+                right: parent.right
+            }
+
+            text: manager.getExampleData()
+            readOnly: true
+            textFormat: TextEdit.PlainText
+        }
+    }
+
+    Column
+    {
+        id: radioButtonsRow
+        width: parent.width
+        anchors.bottom: buttonRow.top
+        anchors.bottomMargin: UM.Theme.getSize("default_margin").height
+
+        ExclusiveGroup { id: group }
+
+        RadioButton
+        {
+            id: dontSendButton
+            text: catalog.i18nc("@text:window", "I don't want to send these settings")
+            exclusiveGroup: group
+            onClicked:
+            {
+                baseDialog.allowSendData = !checked;
+            }
+        }
+        RadioButton
+        {
+            id: allowSendButton
+            text: catalog.i18nc("@text:window", "Allow sending these settings to improve Cura")
+            exclusiveGroup: group
+            onClicked:
+            {
+                baseDialog.allowSendData = checked;
+            }
+        }
+    }
+
+    Item
+    {
+        id: buttonRow
+        anchors.bottom: parent.bottom
+        width: parent.width
+        anchors.bottomMargin: UM.Theme.getSize("default_margin").height
+
+        UM.I18nCatalog { id: catalog; name: "cura" }
+
+        Button
+        {
+            anchors.right: parent.right
+            text: catalog.i18nc("@action:button", "Ok")
+            onClicked: {
```

Comment:
Code style: Brackets on new line.

Reply:
Fixed in 4d3acddfa4b434165c52e99f2d0d9969b614edbe